Dockerfile

```diff
@@ -1,16 +1,18 @@
-FROM okteto/okteto:1.13.2 as okteto
+FROM okteto/okteto:latest as okteto
```

Comment:
Do we want to use latest tag? I'm not sure, but I think Github builds the image each time an action is triggered. It could happen that users using a specific version of the GitHub action has different behaviors because the okteto version changes between executions

Reply:
Agree with Nacho, I think it's better if we pin a version.

Reply:
We introduced a fix to the latest release to set the sourceUrl on preview envs, it will be back to `1.13.2` tomorrow after the community call